superagi/tools/email/send_email_attachment.py

```diff
@@ -94,8 +97,13 @@ def send_email_with_attachment(self, to, subject, body, attachment_path, attachm
             if ctype is None or encoding is not None:
                 ctype = "application/octet-stream"
             maintype, subtype = ctype.split("/", 1)
-            with open(attachment_path, "rb") as file:
-                message.add_attachment(file.read(), maintype=maintype, subtype=subtype, filename=attachment)
+            if StorageType.get_storage_type(get_config("STORAGE_TYPE", StorageType.FILE.value)) == StorageType.S3:
+                attachment_data = S3Helper().read_binary_from_s3(attachment_path)
+            else:
+                with open(attachment_path, "rb") as file:
+                    attachment_data = file.read()
+            attachment_data = S3Helper().read_binary_from_s3(attachment_path)
```

Comment:
why two times  we have  attachment_data = S3Helper().read_binary_from_s3(attachment_path)